admin/src/pages/Discover/index.tsx

```diff
@@ -80,138 +81,150 @@ const Discover = ({ config }) => {
 
   const {
     isLoading: isLoadingForData,
-    data: { result, pagination = {} },
+    data: {result, pagination = {}},
     isFetching,
   } = useQuery(
-    ["get-data", queryParams, canAccess],
+    ['get-data', queryParams, canAccess],
     () => fetchData(queryParams, toggleNotification),
     {
       initialData: {},
-    }
+    },
   );
 
   const handleClickDisplay = (id) => {
     push(getUrl(`discover/${id}`));
   };
 
   const isLoading = isLoadingForData || isFetching;
-  const { total } = pagination;
+  const {total} = pagination;
 
   const emptyLayout = {
     comments: {
-      id: getMessage("page.discover.table.empty"),
+      id: getMessage('page.discover.table.empty'),
       defaultMessage: "You don't have any comments yet.",
     },
     search: {
-      id: getMessage("page.discover.table.empty.search"),
-      defaultMessage: "No comments match the search.",
+      id: getMessage('page.discover.table.empty.search'),
+      defaultMessage: 'No comments match the search.',
     },
   };
 
-  const emptyContent = _q ? "search" : "comments";
+  const emptyContent = _q ? 'search' : 'comments';
 
   const COL_COUNT = 8;
```

Comment:
Can we move it outside of the component?

Reply:
This values comes from useQueryParams hook